Cue/app/tabs/library/LibraryHome.js

```diff
@@ -81,9 +85,28 @@ class LibraryHome extends React.Component {
       refreshing,
     }
   }
+
+  componentWillReceiveProps(newProps: Props) {
+    if (newProps.inaccessibleDecks && newProps.inaccessibleDecks.length > 0) {
+      newProps.inaccessibleDecks.forEach(deck => {
+        Alert.alert(
+          (Platform.OS === 'android' ?
+            'This deck is no longer available from the original owner' :
+            'This Deck Is No Longer Available From The Original Owner'),
+            'To continue using the deck “' + deck.name + '”, copy it into your library.',

```

Comment:
Nice move 👍 Since the deck name is moved into the alert body, you should probably replace "This" in the title with "A" or just remove it completely. Also `from the`!